Makefile

```diff
@@ -482,7 +484,15 @@ govulncheck:
 vulncheck: ## Run govulncheck to find vulnerabilities in code
 	@./scripts/vulncheck.sh ./vuln-ignore
 
+envsubst:
+	@which envsubst || go install github.com/drone/envsubst/cmd/envsubst@latest
+
+docker-sbom:
+	@docker sbom --help > /dev/null || \
+	DOCKER_SBOM_PLUGIN_VERSION=$(DOCKER_SBOM_PLUGIN_VERSION) \
+	ARCH=$(TARGET_ARCH) OS=$(TARGET_OS) ./scripts/install-docker-sbom-plugin.sh
```

Comment:
Let's not automate this. We can spit out a message "please install the docker sbom plugin" and point to the script.

Reply:
I have replaced with a help message pointing to the docs, and the docs explain how to install and link to the script to be used manually if one prefers that.